README.md

```diff
@@ -102,14 +117,19 @@ Create/load measures units:
     mi, err := stats.NewMeasureInt64("/my/otherName", "some other measure")
     if err != nil {
         // handle error
-    }    
+    }
     ...
 
+TODO: where to specify "unit"? Shouldn't it be consistent with
```

Comment:
The unit is actually just a string that obeys the rules defined here http://unitsofmeasure.org/ucum.html
The proto you are referring to is wrong/old. @dinooliva can provide more context if needed about the units.

Reply:
@bogdandrutu also told me the proto is outdated and soon will be updated. removed.